flocker/node/agents/blockdevice.py

```diff
@@ -1121,8 +1121,10 @@ def list_live_nodes():
         This is used to figure out which nodes are dead, so that other
```

Comment:
I do think we need to rewrite this before it's public. I'll add a comment to that effect.